src/platform/intel/cavs/platform.c

```diff
@@ -306,14 +306,18 @@ int platform_boot_complete(uint32_t boot_message)
 
 	mailbox_dspbox_write(0, &ready, sizeof(ready));
 
+#if CONFIG_IPC_MAJOR_3
```

Comment:
Why do we need IPC3 for Intel on main branch ? Lets just remove all IPC3 for all Intel code.